SConstruct

```diff
@@ -129,7 +129,8 @@ opts.Add(BoolVariable("verbose", "Enable verbose output for the compilation", Fa
 opts.Add(BoolVariable("progress", "Show a progress indicator during compilation", True))
 opts.Add(EnumVariable("warnings", "Level of compilation warnings", "all", ("extra", "all", "moderate", "no")))
 opts.Add(BoolVariable("werror", "Treat compiler warnings as errors", not env_base.stable_release))
-opts.Add(BoolVariable("dev", "If yes, alias for verbose=yes warnings=extra werror=yes", False))
+opts.Add(BoolVariable("dev", "If `yes`, alias for verbose=yes warnings=extra werror=yes", False))
+opts.Add(BoolVariable("release", "If `yes`, alias for target=release[_debug] use_lto=yes debug_symbols=no", False))
```

Comment:
`alias for target=release[_debug]` is confusing. I say just make `target=release` fallback to `target=release_debug` with tools builds and change the description here to just `alias for target=release`.